Makefile

```diff
@@ -113,7 +113,7 @@ format-cpp:
 	./bin/format $(CHECK) ./pennylane_lightning
 
 format-python:
-	isort --profile black ./pennylane_lightning/ ./mpitests ./tests $(CHECK)
+	isort --py 311 --profile black -l 100 -p pennylane_lightning ./pennylane_lightning/ ./mpitests ./tests $(CHECK)
```

Comment:
So with this, we can fix the default to follow python3.11 version, right?

Reply:
We needed all those options before things stabilized. I think the main issue was actually `-p pennylane_lightning` because sometimes it would be recognize as first party and others third party.

Reply:
Interesting. Thanks!